### PR TITLE
[4.4] update to 5

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #41125 .

### Summary of Changes
delete if exists

### Testing Instructions

see #41125

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 1275fd246d0c0b616fecd6c4afa98c46299a0132<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41127">issues.joomla.org/tracker/joomla-cms/41127</a>.</sub>

**Comment by richard67** [comment]
@alikon The update succeeds with this PR, but I still get a PHP notice:
```
PHP Notice:  Indirect modification of overloaded property Joomla\\CMS\\Http\\Response::$body has no effect in /home/richard/lamp/public_html/joomla-cms-4.4-dev/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php on line 498
```
I think it needs a similar modification for that code line here https://github.com/joomla/joomla-cms/blob/1275fd246d0c0b616fecd6c4afa98c46299a0132/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L498 as you have done with your other PR #41127 for the CLI here: https://github.com/joomla/joomla-cms/blob/46534b97c672fc6d96516d533d7ff8c2c9eed79e/libraries/src/Installer/InstallerHelper.php#L121-L125

Would it make sense to fix that with this PR here, too?

**Comment by alikon** [comment]
done

File: administrator/components/com_joomlaupdate/src/Model/UpdateModel.php [hunk 1]
```diff
@@ -492,8 +494,11 @@ protected function downloadPackage($url, $target)
             return false;
         }
 
+        // Fix Indirect Modification of Overloaded Property
+        $body = $response->body;
```
Comment (richard67) [inline administrator/components/com_joomlaupdate/src/Model/UpdateModel.php]:
```suggestion
        $body = $result->body;
```
Here it is `$result`.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 3309ffc78250fdb3517b0178b6b9e39034ae9220<br /><br />Hint for other testers: For testing the fix I've updated from a 4.4-dev with this PR applied to a 5.0-dev package where this PR has been merged into.

You can find such a package at https://test5.richard-fath.de/Joomla_5.0.0-alpha3-dev-Development-Update_Package_2023-07-08_test-pr-41127.zip , and you can use the custom update URL https://test5.richard-fath.de/next_major_list_test-pr-41127.xml to update to that package.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41127">issues.joomla.org/tracker/joomla-cms/41127</a>.</sub>

**Comment by richard67** [comment]
@brianteeman Could you repeat your test with the latest changes? Thanks in advance.

**Comment by alikon** [comment]
@MacJoom thanks

**Pull request merged**